entrypoint.sh

```diff
@@ -9,6 +9,11 @@ if [[ "$INPUT_ADOC_FILE_EXT" != .* ]]; then
     INPUT_ADOC_FILE_EXT=".$INPUT_ADOC_FILE_EXT";
 fi
 
+# Install git-lfs since it is not in Alpine base image.
```

Comment:
I think the second option you mentioned in #36 is better, since we can provide a docker image with everything installed and reduce the code of `entrypoint.sh`.

The `apk add openssh-client` in that script could even be moved there too, but that is another issue.